pontoon/base/models.py

```diff
@@ -25,12 +25,72 @@
 log = logging.getLogger('pontoon')
 
 
+class UserTranslationsManager(models.Manager):
+    """
+    Provides various method to interact with larger sets of translations and their stats for user.
+    """
+
+    def _changed_translations_count(self, args):
+        """
+        Helper method, returns expression object which allows us to annotate querysets
+        with counts of translations.
+        """
+        translation_query = (
+            ~Q(translation__string=F('translation__entity__string'))
+            & ~Q(translation__string=F('translation__entity__string_plural'))
+            & Q(translation__user__isnull=False)
+        )
+
+        if args:
+            translation_query &= args
```

Comment:
I don't think this is valid (does this work when you test locally? I'd be really surprised if it did).

Sorry if I was misleading before. What I was trying to convey was that if the arguments to this function are `self, *args` (which I think they should be), then you can change this block to be:

```python
for q_filter in args:
    translation_query &= q_filter
```

Because `*args` as a parameter means the value of `args` within the function is always a list, you can loop over it. And it makes calling this function cleaner and more convenient because you don't have to wrap the arguments in a list manually.